kedro/extras/datasets/tensorflow/tensorflow_model_dataset.py

```diff
@@ -25,14 +25,28 @@ class TensorFlowModelDataset(AbstractVersionedDataSet[tf.keras.Model, tf.keras.M
     The underlying functionality is supported by, and passes input arguments through to,
     TensorFlow 2.X load_model and save_model methods.
 
+    .. code-block:: yaml
+
+        >>> tensorflow_model:
+        >>>   type: tensorflow.TensorFlowModelDataset
+        >>>   filepath: data/06_models/tensorflow_model
+        >>>   load_args:
+        >>>     compile: False
+        >>>   save_args:
+        >>>     overwrite: True
+        >>>     include_optimizer: False
+        >>>   credentials: tf_creds
+        >>>
+
+
     Example:
     ::
 
         >>> from kedro.extras.datasets.tensorflow import TensorFlowModelDataset
         >>> import tensorflow as tf
         >>> import numpy as np
         >>>
-        >>> data_set = TensorFlowModelDataset("saved_model_path")
+        >>> data_set = TensorFlowModelDataset("saved_model/my_model")
```

Comment:
```suggestion
        >>> data_set = TensorFlowModelDataset("data/06_models/tensorflow_model.h5")
```

Looks like the default is a `h5` file, would need to update the `filepath` in the YAML as `h5` as well.

https://www.tensorflow.org/api_docs/python/tf/keras/models/save_model